assets/vendor/polyfill.js

```diff
@@ -2,9 +2,14 @@ if (typeof setTimeout === 'undefined') { var setTimeout = function setTimeout(fu
 
 if (typeof window === 'undefined') {
   window = {
+    addEventListener: function() {},
+    matchMedia: function() {
```

Comment:
i think this can just be matchMedia: function()  { return {} },

Reply:
Fails on Chrome if removed.

Reply:
that doesn't make sense, this should never run in chrome

Reply:
GoogleChrome the browser. It works with Firefox though.

Reply:
Done.